MAUI/Scheduler/reminders.md

```diff
@@ -0,0 +1,423 @@
+---
+layout: post
+title: Appointments Reminders in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about the appointments reminders support in Syncfusion .NET MAUI Scheduler (SfScheduler) appointments.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Appointments reminders in .NET MAUI Scheduler (SfScheduler)
+
+The MAUI Scheduler notify an appointment reminders by using  the [EnableReminder] property and [ReminderAlertOpening] event. An appointment can have one or more reminders.
```

Comment:
MAUI  -> use the Term ".Net MAUI" - correct elsewhere

Reply:
Updated as ".Net MAUI".